corehq/sql_accessors/sql_templates/get_case_ids_in_domain.sql

```diff
@@ -1,16 +1 @@
 DROP FUNCTION IF EXISTS get_case_ids_in_domain(TEXT, TEXT);
```

Comment:
this makes sense retroactively, but do we not also want to delete the function in a migration that runs after this commit?